covasim/webapp/cova_app.js

```diff
@@ -244,6 +260,16 @@ var vm = new Vue({
             this.intervention_figs = response.data;
         },
 
+        parse_day(day) {
+            if (day == null || day == '') {
```

Comment:
This should be '===' to prevent type coercion.

Reply:
I think we want type coercion here actually!

Reply:
I don't agree, but resolving.